lib/utils/fs.go

```diff
@@ -35,9 +33,9 @@ import (
 // It also makes sure that base dir exists
 func EnsureLocalPath(customPath string, defaultLocalDir, defaultLocalPath string) (string, error) {
 	if customPath == "" {
-		homeDir := getHomeDir()
-		if homeDir == "" {
-			return "", trace.BadParameter("no path provided and environment variable %v is not not set", teleport.EnvHome)
+		homeDir, err := os.UserHomeDir()
+		if err != nil || homeDir == "" {
```

Comment:
nit:
I don't think that `os.UserHomeDir()` can return an empty path, but I guess it doesn't hurt to leave it. https://github.com/golang/go/blob/19309779ac5e2f5a2fd3cbb34421dafb2855ac21/src/os/file.go#L507